include/dca/parallel/util/get_workload.hpp

```diff
@@ -42,6 +42,34 @@ int getWorkload(const unsigned int total_work, const unsigned int n_local_worker
   return getWorkload(local_work, n_local_workers, local_id);
 }
 
+inline void getComputeRange(const int& my_rank, const int& mpi_size,
```

Comment:
unit test

Reply:
Agreed. I will prepare one soon.

Reply:
I also believe there is an off by one error here. I will add a PR for you to look at @weilewei

Reply:
PR filed against your distG4_pr on STELLAR-GROPU:distG4_pr

Reply:
Thank you!